src/EPR.Accreditation.Facade/Services/AccreditationService.cs

```diff
@@ -255,5 +255,35 @@ private Common.Dtos.Accreditation CreateNewAccreditationDto(OperatorType operato
 
             return taskProgress;
         }
+
+        public async Task<OverseasReprocessingSiteOutputs> GetOverseasReprocessingSiteOutputs(
+            Guid accreditationExternalId,
+            Guid overseasSiteExternalId)
+        {
+            var overseasSite = await _httpAccreditationService.GetOverseasReprocessingSite(accreditationExternalId, overseasSiteExternalId);
+            if (overseasSite == null)
+            {
+                throw new Exception($"Over seas site with id {overseasSiteExternalId} not found.");
+            }
+
+            var overseasSiteOutputs = _mapper.Map<OverseasReprocessingSiteOutputs>(overseasSite);
+            return overseasSiteOutputs;
+        }
+
+        public async Task UpdateOverseasReprocessingSiteOutputs(
+            Guid accreditationExternalId,
+            OverseasReprocessingSiteOutputs overseasSiteOutputs)
+        {
+            var overseasSite = await _httpAccreditationService.GetOverseasReprocessingSite(accreditationExternalId, overseasSiteOutputs.ExternalId.Value);
```

Comment:
We should be able to perform an update without retrieving the details from the backend first. We should be looking at just sending the updates across to the back end. And the backend should be able to deal with it

Reply:
I thought the whole point was to have a limited number of endpoints in the back end API. There are also singular Update methods in the API, for the various controllers. 

I can go ahead and start creating "new" PUT or PATCH endpoints in the back end, but keep in mind that will create new code in the service class as well. 

Whichever way you wanna go with it?

Reply:
No, i don't mean a new endpoint. It's just that you are doing a GET from the API, only to then do an Update on the data. We shouldn't need to do the get. I had done that myself in a couple of places, but have now removed the calls.